src/test/java/LocationsUtilityTest.java

```diff
@@ -106,4 +107,29 @@ public void getMultipleUniqueLocations() {
         LocationsUtility.getLocations(
             PREFIX, ImmutableList.of(TASK_WITH_LOCATION_ONE, TASK_WITH_LOCATION_TWO)));
   }
+
+  @Test
+  public void getCombinationsNoLists() {
+    List<List<String>> noLists = ImmutableList.of();
+    List<List<String>> actual = LocationsUtility.generateCombinations(noLists);
+    Assert.assertEquals(ImmutableList.of(ImmutableList.of()), actual);
```

Comment:
nit: `actual.isEmpty()`

Reply:
I can't do `actual.isEmpty()` for that because the nested list makes the list not empty but I managed to change this for two of the other tests.